src/_pytest/fixtures.py

```diff
@@ -165,22 +171,24 @@ def getfixturemarker(obj: object) -> FixtureFunctionMarker | None:
 
 
 @dataclasses.dataclass(frozen=True)
-class FixtureArgKey:
+class ParamArgKey:
+    # The param name.
```

Comment:
It's not entirely clear if this is meant as a docstring or a comment for the following line. I'd suggest adding an explicit docstring and putting the comment on the same line as what it refers to.

Reply:
Alternatively, we could make use of attribute docstrings.

Reply:
> Alternatively, we could make use of attribute docstrings.

I agree, that's what we have been using in general I believe (to be clear I'm talking about using `#:` comments in the line above the attribute declaration).